back/src/adapters/secondary/pg/repositories/PgAgencyGroupRepository.integration.test.ts

```diff
@@ -0,0 +1,94 @@
+import { Pool } from "pg";
+import { AgencyDtoBuilder, AgencyGroup, expectToEqual } from "shared";
+import { KyselyDb, makeKyselyDb } from "../kysely/kyselyUtils";
+import { getTestPgPool } from "../pgUtils";
+import { PgAgencyGroupRepository } from "./PgAgencyGroupRepository";
+import { PgAgencyRepository } from "./PgAgencyRepository";
+
+const agency = new AgencyDtoBuilder().build();
+
+describe("PgAgencyGroupRepository", () => {
+  let pool: Pool;
+  let transaction: KyselyDb;
+  let pgAgencyGroupRepository: PgAgencyGroupRepository;
+  let pgAgencyRepository: PgAgencyRepository;
+
+  beforeEach(async () => {
+    pool = getTestPgPool();
+    transaction = makeKyselyDb(pool);
+
+    await transaction.deleteFrom("agency_groups__agencies").execute();
+    await transaction.deleteFrom("agency_groups").execute();
```

Comment:
supprimer aussi les agences 

```suggestion
    await transaction.deleteFrom("agency_groups__agencies").execute();
    await transaction.deleteFrom("agency_groups").execute();
    await transaction.deleteFrom("agencies").execute();
```